ADS v2.8.6 release notes and bump version (#228)
oracle/accelerated-data-science: docs/source/release_notes.rst

Release Notes
=============

2.8.6
-----
Release date: June 13, 2023

> * Resolved an issue in the ``ads opctl build-image job-local``, where the building of job-local would become stuck. Updated the python version to 3.8 in the base environment of the ``job-local`` image.

Reviewer: ```suggestion
* Resolved an issue in ``ads opctl build-image job-local`` when the build of ``job-local`` would get stuck. Updated the Python version to 3.8 in the base environment of the ``job-local`` image.
```

> * Fixed a bug that prevented the support of defined tags for Data Science Job runs.

Reviewer: ```suggestion
* Fixed a bug that prevented the support of defined tags for Data Science job runs.
```

* Fixed a bug in the ``entryscript.sh`` of ``ads opctl`` that attempted to create a temporary folder in the ``/var/folders`` directory.
* Added support for defined tags in the Data Flow application and application run.

> * Deprecated the old :py:class:`~ads.model.ModelDeploymentProperties`, :py:class:`~ads.model.ModelDeployer`  classes, and their corresponding APIs.

Reviewer: ```suggestion
* Deprecated the old :py:class:`~ads.model.ModelDeploymentProperties` and  :py:class:`~ads.model.ModelDeployer`  classes, and their corresponding APIs.
```

* Enabled the uploading of large size model artifacts for the :py:class:`~ads.model.ModelDeployment` class.

> * Implemented validation for shape name and shape configuration details in Data Science Jobs and Data Flow Applications.

Reviewer: ```suggestion
* Implemented validation for shape name and shape configuration details in Data Science jobs and Data Flow applications.
```

> * Added the capability to create ``ADSDataset`` through the pandas accessor.

Reviewer: ```suggestion
* Added the capability to create ``ADSDataset`` using the Pandas accessor.
```

* Made ``Docker`` dependency optional for ``ads opctl run``.
* Provided a prebuilt watch command for monitoring Data Science Jobs with ``ads opctl``.

> * Eliminated the legacy ``ads.dataflow`` package from the ADS code base.

Reviewer: ```suggestion
* Eliminated the legacy ``ads.dataflow`` package from ADS.
```
